packages/pwa-kit-dev/bin/pwa-kit-dev.js

```diff
@@ -291,6 +313,84 @@ const main = () => {
             )
         })
 
+    managedRuntimeCommand('logs')
+        .description(`display environment logs (without --tail, displays this help text and exits)`)
+        .addOption(
+            new program.Option('-p, --project <projectSlug>', 'the project slug').default(
+                null,
+                "the 'name' key from package.json"
+            )
+        )
```

Comment:
I find this help text kind of horrendous! Do we have a plan to support `pwa-kit-dev logs` without `--tail` soon? You know, in the next few months?

I suspect we don't and we'll be looking at this help text in a year thinking "weird, who thought that was a good idea?"!

Can we do one of:

1. Rename the command to `tail-logs`.
2. Remove the `--tail` flag and just tail when you run `pwa-kit-dev logs`, the only thing we have planned.
3. Keep `--tail`, but provide some behaviour for `pwa-kit-dev logs` right away? You could show the first page of logs and then exit, for instance.

Reply:
If you call it `tail-logs` I think there's an easy way to build a `logs` command in future, while maintaining backwards compatibility with an alias...

Reply:
Ok, I'm convinced. I went with option 1.